quickwit/quickwit-cli/src/generate_markdown.rs

```diff
@@ -141,7 +141,7 @@ fn markdown_for_command_helper(
                 "".to_string()
             };
             println!(
-                "`--{}` {}{} \\", // Insert line break after option
+                "`--{}` {}{}  ", // Insert line break after option
```

Comment:
I can't remember the rendering issue we got with using `  ` as a line break, maybe something related to Docusaurus, but we ended up using `\` to fix it.

Reply:
Spoke with @guilload about it. That solution works but it involves trailing spaces, which some editors remove, which will most likely cause this issue to come back eventually. I will try to convert options to a table or some other more reliable structure as a follow up.